.travis.yml

```diff
@@ -138,6 +138,9 @@ after_script:
       --include='**/' \
       --include='**/surefire-reports/**.xml' \
       --include='**/surefire-reports/emailable-report.html' \
+      --include='**/test-reports/junitreports/**.xml' \
+      --include='**/test-reports/emailable-report.html' \
```

Comment:
Also - just to make sure - is it needed because that's where the product tests put test reports?